source/_components/orangepi_gpio.markdown

```diff
@@ -0,0 +1,130 @@
+---
+layout: page
+title: "Orange Pi GPIO"
+description: "Instructions on how to integrate the GPIO capability of a Orange Pi into Home Assistant."
+date: 2019-03-29 19:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category:
+  - DIY
+  - Binary Sensor
+  - Cover
```

Comment:
If you remove the cover platform, then also here 😉

Reply:
That's sloppy, thank you.